### DISCUSSION TITLE
demo not allowing to login

### DISCUSSION THREAD

**Discussion by swarakaka** [orchidsoftware/platform#2185] [General]
‌Hello Administrators,

Always return the following message when you sign in. Demo data may not match those based on it.

**The details you entered did not match our records. Please double-check and try again.**

**Comment by AlejandroAkbal** [comment]
Have you created an admin user?

**Reply by swarakaka** [reply]
Sorry, i meant here:

https://demo.orchid.software/login

**Comment by tabuna** [comment]
Hi @jprobbe, Could you take a look?

**Comment by jprobbe** [comment]
Fixed :
***@***.***

De : Alexandr Chernyaev ***@***.***>
Répondre à : orchidsoftware/platform ***@***.***>
Date : mercredi 23 mars 2022 à 13:13
À : orchidsoftware/platform ***@***.***>
Cc : Jean-Philippe ROBBE ***@***.***>, Mention ***@***.***>
Objet : Re: [orchidsoftware/platform] demo not allowing to login (Discussion #2185)


Hi @jprobbe<https://github.com/jprobbe>, Could you take a look?
